src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const heading = document.querySelector('thead').firstElementChild;
+const headingItems = [...heading.querySelectorAll('th')];
+const tableBody = document.querySelector('tbody');
+const tableRows = [...tableBody.children];
+
+headingItems.forEach(item => item.setAttribute('class', 'column'));
+
+function tableSort(column) {
+  const index = headingItems.indexOf(column);
+
+  const callbackSort = (rowA, rowB) => {
+    const first = rowA.children[index].innerText;
+    const second = rowB.children[index].innerText;
+
+    if (index <= 1) {
+      return first.localeCompare(second);
+    } else if (index === 3) {
+      return first.slice(1).split(',').join('')
```

Comment:
Create function for transforming this type of string into number as you repeat it two times at least.